clients/src/main/java/org/apache/kafka/common/errors/InvalidAclException.java

```diff
@@ -6,33 +6,30 @@
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package kafka.security.auth
 
-import kafka.common.{KafkaException}
-import org.junit.{Test, Assert}
-import org.scalatest.junit.JUnitSuite
+package org.apache.kafka.common.errors;
 
-class OperationTest extends JUnitSuite {
+/**
```

Comment:
I am not sure why Git is marking this as a Rename. It makes review difficult. @SinghAsDev Can you see if this can be fixed? Perhaps squashing the commits would help?